assets/js/modules/analytics-4/components/setup/SetupForm.test.js

```diff
@@ -96,8 +122,6 @@ describe( 'SetupForm', () => {
 			.dispatch( MODULES_ANALYTICS_4 )
 			.selectAccount( accountID );
 
-		registry.dispatch( MODULES_ANALYTICS_4 ).setAccountID( accountID );
```

Comment:
I cleaned up all of these where it was called after `selectAccount` which uses this internally and thus redundant.